app/controllers/mixins/ems_common_angular.rb

```diff
@@ -346,6 +354,7 @@ def build_connection(ems, default_endpoint, amqp_endpoint, ssh_keypair_endpoint,
 
       ems.connection_configurations=([{:endpoint => default_endpoint, :authentication => default_authentication},
                                       {:endpoint => amqp_endpoint, :authentication => amqp_authentication},
+                                      {:endpoint => ceilometer_endpoint, :authentication => ceilometer_authentication},
```

Comment:
hm, I think it should be best solved on this level, for the Endpoint alternatives, we should send only endpoints that were present in the Form and we should delete rest.

So either here, or inside of this method, I need to try it.